lib/Deserializer/functions.php

```diff
@@ -245,3 +245,49 @@ function repeatingElements(Reader $reader, string $childElementName) : array {
     return $result;
 
 }
+
+/**
+ * This deserializer helps you to deserialize structures which contain mixed content like this:
+ * 
+ * <p>some text <extref href="..">and a inline tag</extref>and even more text</p>
+ *
+ * The above example will return
+ *
+ * [
+ *     'some text',
+ *     'and a inline tag',
+ *     'and even more text'
+ * ]
```

Comment:
I  don't think this documentation is correct. It would  also contain elements, no?